AutocompleteClient/FW/UI/ViewModel/AutocompleteViewModel.swift

```diff
@@ -8,7 +8,22 @@
 
 import UIKit
 
-public class AutocompleteViewModel {
+public protocol AbstractAutocompleteViewModel{
```

Comment:
should this protocol be in a separate file?

Reply:
Done.